webapp/src/extensionsBrowser.tsx

```diff
@@ -448,18 +448,26 @@ export const ExtensionsBrowser = (props: ExtensionsProps) => {
                         }
                         {displayMode == ExtensionView.Tabbed &&
                             <div className="tab-header">
-                                <Button
-                                    title={lf("Recommended")}
-                                    label={lf("Recommended")}
-                                    onClick={() => { setCurrentTab(TabState.Recommended) }}
-                                    className={currentTab == TabState.Recommended ? "selected" : ""}
-                                />
-                                <Button
-                                    title={lf("In Development")}
-                                    label={lf("In Development")}
-                                    onClick={() => { setCurrentTab(TabState.InDevelopment) }}
-                                    className={currentTab == TabState.InDevelopment ? "selected" : ""}
-                                />
+                                {extensionsInDevelopment.length > 0 &&
```

Comment:
nit: `!!extensionsInDevelopment?.length &&` is my fav expression because it defends against null refs

Reply:
I use `!!` as recommended since an non undefined/null/zero length is what we want. I use just `!` in the other case since we don't want to show tabs if length is undefined/null/zero.